backend-docker/config.json

```diff
@@ -4,7 +4,7 @@
 			"Endpoint" : "unix:///var/run/docker.sock"
 		},
 		{
-			"UseBoot2Docker": false,
+			"UseBoot2Docker": true,
```

Comment:
Was it causing the issue I am having?

Failed to make new docker client using configuration:
	unknown -- Using Boot2Docker with its environment variables: $DOCKER_HOST and $DOCKER_CERT_PATH
Reason:empty docker configuration